src/services/refactors/moveToFile.ts

```diff
@@ -1104,7 +1104,7 @@ function moveStatementsToTargetFile(changes: textChanges.ChangeTracker, program:
     }
 }
 
-function getOverloadRangeToMove(sourceFile: SourceFile, statement: Statement) {
+function getOverloadRangeToMove(sourceFile: SourceFile, statement: Statement | undefined) {

```

Comment:
`statement === undefined` gets passed here in the test run so I adjusted this to match the reality more closely